integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/authorization/DatalakeDatahubCreateAuthTest.java

```diff
@@ -81,11 +81,11 @@ public void testCreateEnvironment(MockedTestContext testContext) {
                 .when(sdxTestClient.createInternal(), key(sdxInternal))
                 .awaitForFlow(key(sdxInternal))
                 .await(SdxClusterStatusResponse.RUNNING)
-                .when(sdxTestClient.describeInternal(), RunningParameter.who(Actor.useRealUmsUser(AuthUserKeys.MGMT_CONSOLE_ADMIN_B)))
+                .when(sdxTestClient.describeInternal(), RunningParameter.who(Actor.useRealUmsUser(AuthUserKeys.MGMT_CONSOLE_ADMIN_B, testContext)))
```

Comment:
Frankly I am not fond of the testContext argument in the test. It feels somehow strange (I'm already calling testContext's method, and shall I pass a testContext). Could we find some way to solve this?

Reply:
Agree, it's ugly but it's working, we should find a better way about this.